src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -514,6 +524,18 @@ private int WriteRunToErrorList(Run run, string logFilePath)
             return sarifErrors.Count;
         }
 
+        private void ShowFilteredSuppressionColumn()
```

Comment:
>Filtered [](start = 25, length = 8)

I haven't found a way to programmatically set a column filter ("Don't show results whose suppression state is 'Suppressed'".

Reply:
Progress: You do it by calling `SetFilter` on an `IWpfTableControl` representing the Error List. Next challenge is how to get the interface.

```
            this.ErrorListTableControl.SetFilter(
                SarifResultTableEntry.SuppressionStateColumnName,
                new ColumnHashSetFilter(
                    suppressionStateColumnDefinition,
                    VSSuppressionState.Suppressed.ToString()));
```

---
In reply to: [537694588](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r537694588) [](ancestors = 537694588)